clerk/organizations.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+
+from clerk import types
+from clerk.client import Service
+
+
+class OrganizationsService(Service):
+    endpoint = "organizations"
+
+    async def list(self) -> List[types.Organization]:
+        """Retrieve a list of all organizations"""
+        async with self._client.get(self.endpoint) as r:
+            s = await r.json()
```

Comment:
```suggestion
            resp_json = await r.json()
```

let's avoid one letter vars